README.md

```diff
@@ -1 +1,84 @@
-# ride-share-two
\ No newline at end of file
+# Ride Share
+Remember the ride share exercise we did with designing and creating a system to track the ride share data from a CSV file? We did a lot of great work on this exercise in creating arrays and hashes of data, but we've learned a lot since we did that exercise! Now, we're going to use our understanding of classes, methods and attributes to create an object-oriented implementation of our ride share system.
+
+## Learning Goals
+Reinforce and practice all of the Ruby and programming concepts we've covered in class so far:
+- Creating and instantiating classes with attributes
+- Creating class and instance methods within our classes
+- Writing pseudocode and creating tests to drive the creation of our code
+- Importing data from a CSV file
+
+## Primary Requirements
+Create a system that will manage information related to ride sharing. The system should have a way to keep track of drivers, riders and the trips that the drivers take with their riders.
+
```

Comment:
Should we add a note that a command-line interface is explicitly not part of the requirements?

We haven't really emphasized CLIs at all since Bank Accounts so maybe they wouldn't think to write that in?